nameparser/config/affixes.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+# https://en.wikipedia.org/wiki/List_of_family_name_affixes
+
+AFFIXES = set([
+    'a',
+    'ab',
+    'af',
+    'av',
+    'ap',
+    'abu',
+    'ait',
+    'aït',
+    'alam',
+    'at',
+    'ath',
+    'aust',
+    'austre',
+    'bar',
+    'bat',
+    'bath',
+    'ben',
+    'bin',
+    'ibn',
+    'bert',
+    'bet',
+    'bint',
+    'da',
+    'das',
+    'de',
+    'degli',
```

Comment:
Many of these affixes are already defined in prefixes.py. 

It seems like mainly what you want is the parts that come before a last name and currently get added to it (wether we call them prefixes or affixes or tussenvoegsels) separately from the last name.